### PR TITLE
Console ui fixes

### PR DESCRIPTION
**Pull request by dcdarrell9**
## Checklist
  - [ ] CHANGELOG.md updated? (if required)

### CODE REVIEW

File: console/templates/partials/navigation_tabs.html [hunk 1]
```diff
@@ -1,32 +1,65 @@
...
+  {% if request.path == '/submit' %}
+    <li id="Submit-active"
+        class="navigation-tabs__item navigation-tabs__item--primary">
+        <a href="/submit"
+           class="btn btn--secondary navigation-tabs__tab"> Submit </a>
+    </li>
+  {% else %}
     <li id="Submit"
-        class="navigation-tabs__item navigation-tabs__item--primary {% if request.path == '/submit' %} navigation-tabs__item--active {% endif %}">
+        class="navigation-tabs__item navigation-tabs__item--primary">
```
Comment (warrenbailey) [inline console/templates/partials/navigation_tabs.html]: These blocks look very similar is possible to put the if statement around the bit that actually changes? I.e. the additional class?

File: console/templates/partials/navigation_tabs.html [hunk 2]
```diff
@@ -1,32 +1,65 @@
...
+  {% if request.path == '/store' %}
+    <li id="Store-active"
+        class="navigation-tabs__item navigation-tabs__item--primary">
+        <a href="/store"
+           class="btn btn--secondary navigation-tabs__tab"> Store </a>
+    </li>
+  {% else %}
     <li id="Store"
-        class="navigation-tabs__item navigation-tabs__item--primary {% if request.path == '/store' %} navigation-tabs__item--active {% endif %}">
+        class="navigation-tabs__item navigation-tabs__item--primary">
```
Comment (warrenbailey) [inline console/templates/partials/navigation_tabs.html]: As above

File: console/templates/partials/navigation_tabs.html [hunk 3]
```diff
@@ -1,32 +1,65 @@
...
+  {% if request.path == '/add_user' %}
+    <li id="Add User-active"
+        class="navigation-tabs__item navigation-tabs__item--primary">
+        <a href="/add_user"
+           class="btn btn--secondary navigation-tabs__tab"> Add User </a>
+    </li>
+  {% else %}
     <li id="Add User"
-        class="navigation-tabs__item navigation-tabs__item--primary {% if request.path == '/add_user' %} navigation-tabs__item--active {% endif %}">
+        class="navigation-tabs__item navigation-tabs__item--primary">
```
Comment (warrenbailey) [inline console/templates/partials/navigation_tabs.html]: and here?

**Review by warrenbailey — CHANGES_REQUESTED** [review]

**Comment by dcdarrell9** [comment]
[sdx-console UI fixes (2)](https://trello.com/c/eDrVTL0o/566-sdx-console-ui-fixes-2)

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**